README.md

```diff
@@ -165,6 +170,7 @@ grunt.initConfig({
     default: {
       options: {
           path: 'test/js/*.html',
+          cwd: 'target/combined-sources',
```

Comment:
If you think the use case that require `cwd` is complex enough to need its own example by all means add one.  Otherwise what is there will suffice :smile: